.neutrinorc.js

```diff
@@ -81,59 +69,42 @@ module.exports = {
             "react/jsx-curly-brace-presence": 0,
             "react/jsx-no-target-blank": 0,
             "object-curly-newline": 0,
-            "react/no-unused-state": 0
+            "react/no-unused-state": 0,
+            "jest/valid-expect": 0,
           }
         }
       }
     ],
+    [
+      '@neutrinojs/web', {
```

Comment:
The `@neutrinojs/react` preset includes web within it, so adding web again here will overwrite the things that the react preset set up. Remove this section and pass the options directly to `@neutrinojs/react` :-) (If there is anything we can do to make this clearer in the docs let me know)

Reply:
I don't think it was due to the docs. I just read your comment in https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/158#pullrequestreview-159908436.
That made me think I needed to add the 'web' preset as well.